live/test/fixtures/expected/cmdline

```diff
@@ -1 +1 @@
- BOOT_IMAGE=/boot/vmlinuz splash=silent mitigations=auto quiet nosimplefb=1
\ No newline at end of file
+ splash=silent mitigations=auto quiet nosimplefb=1
```

Comment:
well, to be sure I would enhance test with various parameters we are using like some rd.kiwi params and some oh others which are more tricky to be sure that regexp matching